src/Bicep.LangServer/Server.cs

```diff
@@ -24,6 +24,9 @@ public class Server : IDisposable
     {
         private readonly OmnisharpLanguageServer server;
 
+        public T? GetService<T>() => server.GetService<T>();
+        public T GetRequiredService<T>() where T : notnull => server.GetRequiredService<T>();
+
```

Comment:
Rather than exposing the DI methods outside of the `Server` class, would it be possible to pass in the initialization configuration as a parameter?

```csharp
public Server(IBicepLangServerOptions options, Action<LanguageServerOptions> onOptionsFunc)
```

And then change
```csharp
.WithServices(RegisterServices);
```
to
```csharp
.WithServices(services => services.AddServerDependencies(config));
```

Reply:
5df10849c